src/window.cpp

```diff
@@ -3478,6 +3478,8 @@ void ReInitAllWindows()
  * @param clss    The class of the window to position.
  * @param setting The actual setting used for the window's position.
  * @return X coordinate of left edge of the repositioned window.
+ *
+ * @todo Replace \c setting magic numbers with an enumeration?
```

Comment:
I don't like todo's like this. I agree that the function would be cleaner without the magic numbers, but it's relatively easy to make lots of todo's like this that don't add any value but distract from the actual code. For small cleanups like this I'd rather leave out the todo than add it.

Reply:
Point taken. The code is probably already littered with too many "TODO" and "XXX" comments, so I'll try to keep them to myself in future. At the very least, we can take solace in the fact that one of the previously documented todos has actually been done :)